framework/src/index.js

```diff
@@ -9,7 +9,5 @@ const validator = require('./controller/helpers/validator');
 module.exports = {
 	Application,
 	version,
-	helpers: {
-		validator,
-	},
+	helpers,
```

Comment:
can we call it utils to align with other libraries?

Reply:
Yes

Reply:
this still need to be resolved

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/3424